admin/acceptance_tests/user_test.py

```diff
@@ -150,3 +166,39 @@ def test_selenium(self, dbsession, selenium):
         dbsession.expire(user)
         assert user.username == 'new_name_éôù'
         assert user.email == 'new_email'
+
+    # in order to make this work, had to install selenium gecko driver
+    @pytest.mark.usefixtures("selenium", "selenium_app")
+    def test_delete_selenium(self, dbsession, selenium):
+        from c2cgeoportal_commons.models.main import User
+        user_id = dbsession.query(User). \
+            filter(User.username == 'babar_13'). \
+            one().id
+        selenium.get('http://127.0.0.1:6543' + '/user/')
+        elem = selenium.find_element_by_xpath("//div[@class='infos']")
+        assert 'Showing 1 to 10 of 23 entries' == elem.text
```

Comment:
is this related with delete ?

Reply:
Ensuring that there 23 entries before delete, then 22 after delete, also that all entries can be read on the same page (50 per pages, i.e. more than 23)